packages/ui-components/src/components/Readme/__snapshots__/Readme.spec.tsx.snap

```diff
@@ -4,55 +4,115 @@ exports[`<Readme /> component should load the component in default state 1`] = `
 {
   "asFragment": [Function],
   "baseElement": .emotion-0 {
-  background: #fff;
-  color: #000;
-  padding: 16px 24px;
+  background-color: #fff;
+  color: rgba(0, 0, 0, 0.87);
```

Comment:
ups sorry, I thought I have removed all this snapshots :( I'll review this again